site/content/en/references/kustomize/kustomization/helmcharts/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Charts"
+linkTitle: "Helm Charts"
+type: docs
+weight: 6
+description: >
+Helm chart inflation generator
+---
+
+[kustomize builtins]: https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_helmchartinflationgenerator_
+[Helm support long term plan]: https://github.com/kubernetes-sigs/kustomize/issues/4401
+
+## Helm Chart Inflation Generator
```

Comment:
This page and/or the builtins reference should probably mention the flags required for this to work